label_studio/feature_flags.json

```diff
@@ -4432,7 +4432,7 @@
     },
     "fflag_fix_front_lsdv_4620_memory_leaks_100723_short": {
       "key": "fflag_fix_front_lsdv_4620_memory_leaks_100723_short",
-      "on": true,
+      "on": false,
```

Comment:
we need to remember to turn this off in LF on delivery

Reply:
Did you find another way to solve memory leaks?

Reply:
There is no memory leaks when we are not building 3 separate React scripts + mobx-state-tree. When bundled together, and served properly this yielded what is expected, that React lifecycles are respected and ultimately MST was not retaining memory. In fact, it now throws errors if running this code because it fundamentally disrupts normal React usage by ripping out the fibers.

Reply:
As I know part of the problem is in React 17 itself.  https://github.com/HumanSignal/label-studio-frontend/pull/1297/files#diff-0a9a22cedc083cfd1838c700569c35f3163ab833e220a32fa42a78432d4427b4R87-R95
If we do not move to React 18 there is no reason to say that there are no memory leaks.